### PR TITLE
Change transaction status and result

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/definitions/Transaction.yaml [hunk 1]
```diff
@@ -21,12 +21,23 @@ properties:
...
+      - waiting-refund
+  result:
+    description: Transaction status
+    type: string
+    readOnly: true
+    enum:
+      - approved
+      - canceled
+      - declined
       - timeout
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: remove timeout

File: spec/definitions/Transaction.yaml [hunk 2]
```diff
@@ -21,12 +21,23 @@ properties:
     type: string
     readOnly: true
     enum:
-      - approved
+      - completed
       - connection-error
       - declined
       - never-sent
-      - pending
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: add pending

Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: (and change this field name to `status`

**Comment by 2e3s** [comment]
Oh, I copied that https://github.com/Rebilly/RebillyReportsAPI/blob/master/spec/definitions/TransactionsPlan.yaml#L42

File: spec/definitions/Transaction.yaml [hunk 3]
```diff
@@ -16,18 +16,28 @@ properties:
...
       - never-sent
       - pending
-      - pending-processor
+      - sending
+      - suspended
       - timeout
+      - waiting-capture
+      - waiting-refund
+  result:
+    description: Transaction status
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: change this to result, and above to status

**Pull request merged**